### DISCUSSION TITLE
[Icon request]  Addition for AWK

### DISCUSSION THREAD

**Discussion by nbehrnd** [ryanoasis/nerd-fonts#926] [Ideas]
I would like to suggest the addition of an icon for AWK.

The code training platform [exercism.org](https://exercism.org/tracks) presents this language with an icon reminiscent to the auk of the seminal book by Aho, Kerningham, an Weinberger shown on the cover (copy on [archive.org](https://archive.org/details/pdfy-MgN0H1joIoDVoIC7/mode/2up)):

![The-AWK-Programming-Language svg](https://user-images.githubusercontent.com/19709084/189400541-9b5833c8-d344-4dad-bb48-813b0ea8bf62.png)

(image credit https://en.wikipedia.org/wiki/AWK#/media/File:The-AWK-Programming-Language.svg)

![awk_exercism](https://user-images.githubusercontent.com/19709084/189400576-4b46152c-ca2e-48dc-abbd-ea5db8f6a78e.png)

(screen photo from  https://exercism.org/tracks)

As by today, a search on https://www.nerdfonts.com/cheat-sheet does not yield an entry relevant to AWK.  Since you set of icons however includes icons with many details e.g., for Rust, I wonder if the addition of a silhouette-like icon of an auk for AWK equally would be possible.

Contrasting to an earlier discussion about [Fortran](https://github.com/ryanoasis/nerd-fonts/discussions/925), I'm not aware if there is an icon in the public domain which may serve as pattern of reference, or the copyright status of the auk on the AWK book.

**Comment by nbehrnd** [comment]
By consultation of the [cheat sheet](https://www.nerdfonts.com/cheat-sheet), an icon was added.  Thank you, I close the ticket.

![awk_icon](https://github.com/user-attachments/assets/86ec3592-f4d5-4a61-a61c-2d3624d0071e)

**Reply by Finii** [reply]
Ah, yes. I did that icon myself and ported it even back upstream.
Glad you like it!

* https://github.com/devicons/devicon/pull/2262
